quickstart/src/main/java/com/microsoft/azure/kusto/quickstart/KustoSampleApp.java

```diff
@@ -360,30 +360,18 @@ private static void alterBatchingPolicy(Client kustoClient, String databaseName,
     }
 
     private static boolean createIngestionMappings(boolean useExistingMapping, Client kustoClient, String databaseName, String tableName, String mappingName, String mappingValue, IngestionProperties.DataFormat dataFormat) {
-        if (!useExistingMapping) {
-            // TODO Deprecated: this restriction is likely to be removed soon
-            if ((dataFormat == IngestionProperties.DataFormat.JSON || dataFormat == IngestionProperties.DataFormat.MULTIJSON || dataFormat == IngestionProperties.DataFormat.SINGLEJSON || dataFormat == IngestionProperties.DataFormat.AVRO) && StringUtils.isBlank(mappingValue)) {
-                System.out.printf("The data format '%s' requires a mapping, but configuration indicates to not use an existing mapping and no mapping was provided. Skipping this ingestion.%n", dataFormat.getKustoValue());
-                return false;
-            }
+        if (!useExistingMapping && StringUtils.isNotBlank(mappingValue)) {

```

Comment:
Nit: you can invert the if and do an early return

Reply:
My thought process was that this structure is more intuitive to understand the intended logic.
Also, it effectively returns early because the implicit `else` immediately returns `true`.